### PR TITLE
Remove unnecesary ember-new-computed addon

### PR DESCRIPTION
**Pull request by cibernox**
If this addons is 1.12+ (and I think that it safe to assume it is), this
addon is not needed anymore.

### CODE REVIEW

**Comment by cibernox** [comment]
Rebased

File: addon/flash/object.js [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-import computed from 'ember-new-computed';
 
 const {
   Object: EmberObject,
   run: { later, cancel },
   Evented,
   get,
-  set
+  set,
+  computed
```
Comment (sbatson5) [inline addon/flash/object.js]: Can we also add `readOnly` like we do with `later` and `cancel`?

```
...
set,
computed,
computed: { readOnly }
```

Comment (cibernox) [inline addon/flash/object.js]: Done.

Comment (sbatson5) [inline addon/flash/object.js]: Ah, it looks we still need to pull in `computed` as well as `computed: { readOnly }`

Comment (cibernox) [inline addon/flash/object.js]: I don't think so. It's not used within that file. This timeout seems unrelated.

Comment (cibernox) [inline addon/flash/object.js]: I'll push again with no changes

Comment (sbatson5) [inline addon/flash/object.js]: Ah, my mistake. Sorry about that.

Comment (cibernox) [inline addon/flash/object.js]: It's green now.

**Comment by cibernox** [comment]
Rebased again.

**Comment by sbatson5** [comment]
This LGTM 👍 . Sorry for the delay in merging!
